app/pages/blackout/QuoteSection/company_name.jsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
```

Comment:
company name should probably just be white and offset from the side with a margin